tests/integration_tests/functional/test_balloon.py

```diff
@@ -120,29 +120,8 @@ def copy_util_to_rootfs(rootfs_path, util):
     subprocess.check_call("rmdir tmpfs", shell=True)
 
 
-# pylint: disable=C0103
-def test_rss_memory_lower(test_microvm_with_ssh_and_balloon, network_config):
+def _test_rss_memory_lower(test_microvm):
     """Check inflating the balloon makes guest use less rss memory."""
```

Comment:
The comment here is not pretty clear to me, as it is the same with the other part on line 156. I think the usages changed a little with the split and this should be specified, since this test would inflate the balloon of a custom provided microVM, while the other "half" will be the complete test run for this usecase. From what I understand, `**_**test_rss_memory_lower` is the one to be called from other tests?

Reply:
This function implements the actual testing part, while the test function just sets up the microvm and then calls _test_rss_memory_lower. The setup part is something that is present in all pytest tests, so it makes sense to comment what the test actually tests for. I think the same comment applies to both.